### DISCUSSION TITLE
Contributor guidelines

### DISCUSSION THREAD

**Discussion by mr-stringer** [sap-linuxlab/community.sap_install#223] [General]
Hi

I'm sure I saw some contributor guidelines for this project, but I can no longer find them.  Can you please let me know where I should look?

Many thanks.

**Comment by berndfinger** [comment]
Here you are:
https://sap-linuxlab.github.io/initiative_contributions/
https://sap-linuxlab.github.io/initiative_contribute/__ansible_dev_guidelines/
